File: CPP/mahasiswa.cpp
#include <bits/stdc++.h>

using namespace std;

class Mahasiswa
{
private:
    string nama, nim, prodi, fakultas;

public:
    // constructor
    Mahasiswa()
    {
        this->nama = "";
        this->nim = "";
        this->prodi = "";
        this->fakultas = "";
    }

    Mahasiswa(string nama, string nim, string prodi, string fakultas)
    {
        this->nama = "";
        this->nim = "";
        this->prodi = "";
        this->fakultas = "";
    }

    string getNama()
    {
        return this->nama;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    string getNIM()
    {
        return this->nim;
    }

    void setNIM(string nim)
    {
        this->nim = nim;
    }

    string getProdi()
    {
        return this->prodi;
    }
    void setProdi(string prodi)
    {
        this->prodi = prodi;
    }

    string getFakultas()
    {
        return this->fakultas;
    }

    void setFakultas(string fakultas)
    {
        this->fakultas = fakultas;
    }
    ~Mahasiswa()
    {
    }
};

File: Java/mahasiswa.java
public class mahasiswa {
    private String nama;
    private String nim;
    private String prodi;
    private String fakultas;

    mahasiswa() {
        this.nama = "";
        this.nim = "";
        this.prodi = "";
        this.fakultas = "";
    }

    mahasiswa(String nama, String nim, String prodi, String fakultas) {
        this.nama = nama;
        this.nim = nim;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setNim(String nim) {
        this.nim = nim;
    }

    public void setProdi(String prodi) {
        this.prodi = prodi;
    }

    public void setFakultas(String fakultas) {
        this.fakultas = fakultas;
    }

    public String getNama() {
        return this.nama;
    }

    public String getNim() {
        return this.nim;
    }

    public String getProdi() {
        return this.prodi;
    }

    public String getFakultas() {
        return this.fakultas;
    }
}
